docs/getting-started/interbtc.md

```diff
@@ -0,0 +1,118 @@
+
+- **Radically Open**. Anyone can become a Vault and help secure interBTC, anytime. Yes, you can run your own Vault!
+
+Hence, as a holder of interBTC, you have the following guarantee:
+
+?> You can always redeem interBTC for BTC, or be reimbursed in the collateral currency at a beneficial rate.
+
+In the a Vault misbehaves, you will be reimbursed from the Vault’s collateral and will end up making a *profitable* trade between BTC and and the collateral asset(s). 
+
+At launch, collateral will be put down in DOT. In the mid/long run, this may be extended to stablecoins or token-sets to improve stability.
+
+Summarizing, to trust interBTC, you only need to:
+
+- *Trust that Bitcoin is secure*. Meaning: trust that Bitcoin blocks are final after X confirmations. The bridge will recommend a minimum of 6 confirmations, though users and apps are encouraged to set higher thresholds.
+- *Trust that Polkadot / the chain you are using interBTC on is secure* . This assumption is made by all applications running on top of Polkadot.
+
+?> interBTC is based on the XCLAIM framework - a peer-reviewed paper published at IEEE S&P, one of the most prestigious security conferences. You can [read the paper](https://eprint.iacr.org/2018/643.pdf) and checkout more of the [research published by the team behind Interlay](../about/research).
+
+
+?> A more in-depth discussion about security can be found in the 200+ page [specification](https://spec.interlay.io/security_performance/liquidations.html).
+
+## Network Participants
+
+XCLAIM’s design has an emphasis on being open and permissionless. As such, any user can take up multiple roles at the same — but also leave the system whenever they wish. As such, to participate in the BTC Parachain, you can choose from:
+
+**Vaults:** collateralized intermediaries who hold BTC locked on Bitcoin. Any user can become a Vault by simply locking DOT collateral. The requirements are (1) a Bitcoin full node, (2) a Polkadot account and (3) liquidity in accepted collateral assets.
+
+**Users:** there are two types of user on the BTC Parachain:
+
+- **Liquidity Providers** lock BTC with Vaults to mint 1:1 backed *interBTC* on the Parachain. Requirement: (1) Bitcoin wallet and (2) Polkadot wallet.
+
+- **End-Users** obtain interBTC from liquidity providers on Polkadot and use interBTC for payments and with applications. Requirements: Polkadot wallet
+
+Both can redeem owned interBTC for BTC at any time (requires a Bitcoin wallet).
+
+
+**Collators** maintain the Interlay parachain by collecting transactions and producing security proofs which are verified by validators of the Polkadot Relay Chain. This is a Polkadot-specific role. [Read more about Collators in the Polkadot Wiki](https://wiki.polkadot.network/docs/learn-collator).
+
+Parachain’s NPoS consensus, as per Polkadot consensus rules. Requirement: (1) Parachain full node and (2) DOTs.
+
+<div id="step-by-step"></div>
+
+## Mint and Redeem
```

Comment:
We should probably agree on a singular terminology at this stage for issue/redeem. We have:
- issue & redeem
- mint & redeem
- wrap & unwrap

wBTC calls it mint & burn. ren calls it mint & release. tbtc calls it mint & redeem.

I think the consensus is to call "issue" "mint" and for "redeem", I think "release" would be good. Burn is misleading since we have it for another process. Redeem might be a bit too complicated of a word rather than "release".

Reply:
I think we need to agree on the terminology first then.